### PR TITLE
nimblenote.nimblenote version 2.0.2.605

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/23722)

### CODE REVIEW

File: manifests/n/nimblenote/nimblenote/2.0.2.605/nimblenote.nimblenote.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: nimblenote.nimblenote
+PackageVersion: 2.0.2.605
```
Comment (OfficialEsco) [inline manifests/n/nimblenote/nimblenote/2.0.2.605/nimblenote.nimblenote.installer.yaml:5]:
```suggestion
PackageVersion: 2.0.2
```

File: manifests/n/nimblenote/nimblenote/2.0.2.605/nimblenote.nimblenote.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: nimblenote.nimblenote
+PackageVersion: 2.0.2.605
+PackageLocale: en-US
+Publisher: nimblenote is a simple keyboard-driven note taking application. Search, create, edit and delete notes all without the mouse.
+PackageName: nimblenote
+License: Copyright © 2020 leaves under foot ltd
+ShortDescription: Simple keyboard-driven note taking
```
Comment (OfficialEsco) [inline manifests/n/nimblenote/nimblenote/2.0.2.605/nimblenote.nimblenote.locale.en-US.yaml:10]:
```suggestion
PackageVersion: 2.0.2
PackageLocale: en-US
Publisher:leaves under foot ltd
PackageName: nimblenote
License: Copyright © 2020 leaves under foot ltd
ShortDescription: Simple keyboard-driven note taking
Description: nimblenote is a simple keyboard-driven note taking application. Search, create, edit and delete notes all without the mouse.
```

File: manifests/n/nimblenote/nimblenote/2.0.2.605/nimblenote.nimblenote.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: nimblenote.nimblenote
+PackageVersion: 2.0.2.605
```
Comment (OfficialEsco) [inline manifests/n/nimblenote/nimblenote/2.0.2.605/nimblenote.nimblenote.yaml:5]:
```suggestion
PackageVersion: 2.0.2
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/n/nimblenote/nimblenote/2.0.2.605/nimblenote.nimblenote.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: nimblenote.nimblenote
+PackageVersion: 2.0.2
```
Comment (vedantmgoyal2009) [inline manifests/n/nimblenote/nimblenote/2.0.2.605/nimblenote.nimblenote.installer.yaml:5]:
```suggestion
PackageVersion: 2.0.2.605

```

File: manifests/n/nimblenote/nimblenote/2.0.2.605/nimblenote.nimblenote.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: nimblenote.nimblenote
+PackageVersion: 2.0.2
```
Comment (vedantmgoyal2009) [inline manifests/n/nimblenote/nimblenote/2.0.2.605/nimblenote.nimblenote.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.0.2.605

```

File: manifests/n/nimblenote/nimblenote/2.0.2.605/nimblenote.nimblenote.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: nimblenote.nimblenote
+PackageVersion: 2.0.2
```
Comment (vedantmgoyal2009) [inline manifests/n/nimblenote/nimblenote/2.0.2.605/nimblenote.nimblenote.yaml:5]:
```suggestion
PackageVersion: 2.0.2.605

```

**Review by vedantmgoyal9 — CHANGES_REQUESTED** [review]
You have not updated the directory name.

File: manifests/n/nimblenote/nimblenote/2.0.2/nimblenote.nimblenote.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: nimblenote.nimblenote
+PackageVersion: 2.0.2
+PackageLocale: en-US
+Publisher:leaves under foot ltd
```
Comment (OfficialEsco) [inline manifests/n/nimblenote/nimblenote/2.0.2/nimblenote.nimblenote.locale.en-US.yaml:7]:
```suggestion
Publisher: leaves under foot ltd
```
My bad
`(Line: 7, Col: 1, Idx: 223) - (Line: 7, Col: 32, Idx: 254): While parsing a block mapping, did not find expected key.`

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**